packages/vector_graphics/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   # See https://github.com/flutter/flutter/issues/157626
-  vector_graphics_compiler: ">=1.1.11+1 <= 1.1.12"
+  vector_graphics_compiler: ">=1.1.11+1 <= 1.1.13"
```

Comment:
1.1.13 doesn't exist; what's the purpose of this change?

Reply:
the same as https://github.com/flutter/packages/pull/8014#discussion_r1827783576